src/mappings/Competition/schema.graphql

```diff
@@ -16,8 +16,10 @@ type CompetitionProposal @entity {
     winningSuggestions: [CompetitionSuggestion!]
     votes: [CompetitionVote!] @derivedFrom(field: "proposal")
     createdAt: BigInt!
-    totalSubmissions: BigInt!
+    totalSuggestions: BigInt!
     totalVotes: BigInt!
+    numWinningSuggestions: BigInt!
```

Comment:
Perhaps to be consistent with other props here, should be `numberOfWinningSuggestions`

Reply:
ok...